subprojects/docs/src/docs/userguide/multiproject.adoc

```diff
@@ -531,15 +531,6 @@ Assume you are working on a single project, the “`:api`” project. You have b
         </sample>
 ++++
 
-While you are working in a typical development cycle repeatedly building and testing changes to the “`:api`” project (knowing that you are only changing files in this one project), you may not want to even suffer the expense of building “`:shared:compile`” to see what has changed in the “`:shared`” project. Adding the “`-a`” option will cause Gradle to use cached jars to resolve any project lib dependencies and not try to re-build the depended on projects.
```

Comment:
:x: For now I would leave this in. We didn't really remove the option yet.

Reply:
Restored.